lib/util/cache/repository/types.ts

```diff
@@ -144,7 +144,13 @@ export interface RepoCacheData {
     gitea?: {
       pullRequestsCache?: GiteaPrCacheData;
```

Comment:
we should also use unknown here and on other places to not have references to platform.

should of cause be done in separate PR